### DISCUSSION TITLE
1st academic conference online reivew

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#9] [General]
다들 책을 잘 읽고 계시고 리뷰 준비도 잘 하고 있으리라 믿고 있습니다.

세 가지 할 얘기가 있는데요

- 적어도 이번 주 화요일인 4월 13일 23:59 까지는 pull request를 만들어 주셔야 이후 3일 간 다른 분들이 comment를 달 시간이 나오게 됩니다.
- 첫 번째 온라인 리뷰는 4월 16일 금요일 오후 11시 입니다.
  - @yoonucho , @hi-hyein 님 에게는 익숙한 시간인데 @Joseph-Cha 님에게는 익숙하지 않은 시간이겠군요
  - 만약 이 시간에 리뷰가 불가능 하신 분은 댓글로 얘기해주셔야 리뷰 일정의 조정이 가능합니다.
- 한 사람당 논의할 내용의 할당 시간은 30분입니다. 자기가 만든 pull request 기준으로 진행합니다.

**Comment by hi-hyein** [comment]
네 잘 준비하고 있습니다!
일부러 막바지 기간에 맞춘건 아니지만;;;; 내일까지 pr올릴 예정입니다😆

**Reply by Joseph-Cha** [reply]
작업 다하고 브랜치를 따려고 보니 쓰기 권한이 없어서 안된다고 하네요...ㅋㅋㅋ🤣

**Reply by jongfeel** [reply]
빠른 권한 부여 해드렸습니다!

**Reply by yoonucho** [reply]
네 저도 정해진 시간안에 올리도록 할게요!

**Comment by jongfeel** [comment]
2nd는 기록에 없지만 아래와 같은 일정으로 진행 되었습니다.

- 4월 27일 까지 chapter04, chapter05 pull request 진행
- 4월 30일 까지 각 pull request 별로 comment 진행
- 4월 30일 오후 11시 2nd academic conference online review 진행

**Comment by jongfeel** [comment]
3rd academic conference online review는 아래의 일정으로 진행합니다.

- 5월 11일 까지 clean coder, chapter 01, 02, 02 진행
- 5월 14일 오후 11시 전 까지 각 pull request 별로 comment 진행 (add review 버튼을 눌러 자신의 review가 진행되도록 해야 함)
- 5월 14일 오후 11시 3rd academic conference online review 진행

입니다.

책 보유 현황은 아래와 같습니다.

- 종필: 새로 주문한 책이 동훈님한테 전달 되면 회사 책으로 진행
- 동훈: 회사 책으로 보다가 새 책이 전달되면 새 책으로 진행
- 혜인: 개인적으로 구매함
- 윤우: 새로 주문한 책이 윤우님에게 전달 예정.
